bundler.d/ovirt.rb

```diff
@@ -1,3 +1,3 @@
 group :ovirt do
-  gem 'rbovirt', '~> 0.0.24'
+  gem 'rbovirt', '~> 0.0.28'
```

Comment:
.32 according to the tags on https://github.com/abenari/rbovirt/commit/e9292f5d8bfb4eb2b11c6323933ebf87349e479f